generators/client/templates/_package.json

```diff
@@ -8,6 +8,7 @@
     "<%= MAIN_SRC_DIR %>bower_components"
   ],
   "devDependencies": {
+    "bower": "1.7.9",
```

Comment:
@deepu105 : Added a CI profile but don't merge yet ...  looking into an issue when no node/npm is installed via the plugin.